_guides/analytics_amp.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: Configure Analytics
+order: 5
+folder: analytics
+---
+
+## Decide before you start
+
+All analytics solutions are built upon knowing what data you need,
+and how you intend to analyze that data. Decide before you start:
+
+* Will you use third-party analytics tools to analyze user engagement,
+or your own in-house solution?
+* What user behaviors will you measure to understand user engagement?
+
+### Send data to vendor or self?
+
+If you have your own in-house solution for measuring user engagement,
+the only thing you will need to integrate AMP analytics with that solution is a URL.
+This is where you will send the data.
+You can also send data to various URLs.
+For example, you can send page view data to one URL,
+and social engagement data to another URL.
+
+AMP analytics is specifically designed to measure once and report to many.
+If you are already working with one or more analytics vendors,
+check the
+[amp-analytics specification](/docs/reference/extended/amp-analytics.html)
+to see if they’ve integrated their solution with AMP.
+If they have, simply link to their docs from the specification
+and start following the instructions.
+
+If the analytics vendor hasn’t integrated with AMP,
+we encourage you to
+[create an issue in the AMP project](https://github.com/ampproject/amphtml/issues/new)
```

Comment:
Also, they should reach out to that vendor to ask for support. Creating the issue is a bit secondary, although it's good for us to know for tracking purposes, which is why I think it's still worth saying they should do it.